Easy/03 - Longest Word.js

```diff
@@ -47,3 +47,13 @@ function LongestWord(sen) {
     }
     return longestWord;
 }
+
+
+// alternative solution of longest word problem (Using Array sort method)
```

Comment:
Please change to be `// Using Array sort() Method`

Reply:
Just pushed by resolving the spacing issue. Please review the code.